test-suite/Makefile

```diff
@@ -104,12 +104,12 @@ PREREQUISITELOG = prerequisite/admit.v.log			\
 #######################################################################
 
 .DELETE_ON_ERROR:
-.PHONY: all run clean $(SUBSYSTEMS)
+.PHONY: all run clean $(SUBSYSTEMS) unit-tests
 
 all: run
 	$(MAKE) report
 
-run: $(SUBSYSTEMS)
+run: $(SUBSYSTEMS) unit-tests
```

Comment:
Why is unit-tests not part of SUBSYSTEMS?

Reply:
I had been treating it specially when there was a separate Makefile. Now that there's just one Makefile, I can put unit-tests into SUBSYSTEMS.